### DISCUSSION TITLE
.gitattributes question

### DISCUSSION THREAD

**Discussion by RedBow-tie** [community/community#137133] [Repositories]
### Select Topic Area

Question

### Body

Hi!

I’ve made a Compiler that’s uses a file extension that is already registered.
I’m trying to move my files to the Other category with

*.sw linguist-language=Other

but it doesn’t work.
Any suggestions ?

**Comment by Yackens** [comment]
To move your files with the .sw extension to the "Other" category in GitHub, you can try the following approach:

First, create a .gitattributes file (if you don't already have one) in the root of your repository.
Then, add the following line to the .gitattributes file:
`*.sw linguist-vendored`

This marks the files as vendored, which typically excludes them from the language statistics. 

Alternatively, to categorize them explicitly as "Other":
`*.sw linguist-language=Other`

`Commit `the changes to your repository.
Force GitHub to reanalyze the repository by performing a `push`, or you can make a small change to the repository to trigger the reanalysis.
Check the results on GitHub after a few minutes to see if the files are categorized correctly.

If you found this answer useful, don't forget to mark it as the accepted answer so more people can see it!

**Comment by RedBow-tie** [comment]
I have already tried to use "*.sw linguist-language=Other" but it didn't work.
linguist-vendored excludes all the files so that can't be used.

**Comment by davevad93** [comment]
Hi @RedBow-tie ,

Since "`Other`" isn't a language per-se but rather a sorting method when a repo has a small percentage of multiple programming languages, I don't know if it can be used in `.gitattributes` with `linguist`, in fact you can't search for "`Other`" in the `Languages` section of the repo. You could try the following but I don't know if it will work:

```git
*.sw linguist-language=Other
*.sw linguist-detectable=true
```

**Reply by RedBow-tie** [reply]
Hi

You where right, it didn’t work.
I suppose, the best thing is to use:  *.sw linguist-detectable=false

Thanks

**Reply by davevad93** [reply]
You're welcome @RedBow-tie , yes I also think that using "`*.sw linguist-detectable=false`" would be the best option.
